traffic_ops/testing/api/v3/withobjs_test.go

```diff
@@ -32,9 +32,15 @@ func WithObjs(t *testing.T, objs []TCObj, f func()) {
 			withFuncs[obj].Delete(t)
 		}
 	}()
-	for _, obj := range objs {
+	var obj TCObj
+	for _, obj = range objs {
 		withFuncs[obj].Create(t)
 	}
+
+	if _, exists := withNegativeFuncs[obj]; exists {
```

Comment:
Do we really need a new negative fixtures json file and all this extra `WithObjs` stuff that goes with it? It seems like unnecessary complexity when negative test data has always been defined in the tests themselves.

Reply:
I can see how generalizing *negative tests* introduces tech debt if I don't move the negative test data to `tc-negative-fixtures.json` at the same time. Since doing that would be outside the scope of this PR:

* Moved the *invalid parameters* data to `CreateNegativeTestParameters()` in c364546143
* Removed the generalized *negative tests* stuff in 04b2fb10ae

It can be added back in a future PR that also moves existing negative test data to the JSON file that the PR would add.